app/src/main/java/org/mozilla/fenix/home/HomeFragment.kt

```diff
@@ -227,11 +213,13 @@ class HomeFragment : Fragment(), CoroutineScope {
                         is SessionControlAction.Collection -> handleCollectionAction(it.action)
                         is SessionControlAction.Onboarding -> handleOnboardingAction(it.action)
                         is SessionControlAction.ReloadData -> {
-                            layoutManagerState?.also { parcelable ->
+                            val homeViewModel = activity?.run {
```

Comment:
@boek question - is there a reason we want to be restoring the layout manager state here if the data is being reloaded?

Reply:
@ekager We have to restore the state after the recyclerview has items, otherwise it's height is `0` and it won't scroll